### PR TITLE
Plone 5.1.x support

### PR DESCRIPTION
**Pull request by maethu**
Affected parts:
- Ressources cachekey (no longer portal_css --> plone.bundles)
- debug mode (no longer css debug_mode --> registry 'plone.resources.development')
- Nav portlet and folder_contents (no longer uses `IContentIcon` adapters --> hardcoded css class)

If the plone 5 Barceloneta theme is applied along with ftw.theming, it results in having two icons in the addable types menu:
screen shot 2017-07-19 at 12 25 35

without theme it's fine:
screen shot 2017-07-19 at 12 26 41

### CODE REVIEW

File: ftw/theming/browser/theming_css.py [hunk 1]
```diff
@@ -12,24 +14,42 @@
...
+
+    if IS_PLONE_5:
+        registry = getUtility(IRegistry)
+        timestamps = []
+        for key in filter(
+                lambda key: (key.startswith('plone.bundles/')
+                             and key.endswith('.last_compilation')),
+                registry.records.keys()):
+            timestamps.append(str(registry[key]))
+            return str(hash(frozenset(timestamps)))
```
Comment (jone) [inline ftw/theming/browser/theming_css.py]: I think this is indented wrong.
⬅️

Comment (maethu) [inline ftw/theming/browser/theming_css.py]: 😱

**Comment by maethu** [comment]
@jone fixes  👍

File: ftw/theming/utils.py [hunk 2]
```diff
@@ -20,3 +24,5 @@ def get_mimetype_css_class_from_icon_path(icon_path):
     return 'icon-mimetype-img-{}'.format(
         normalizer.normalize(image_filename, max_length=255)
     )
+
+
```
Comment (jone) [inline ftw/theming/utils.py]: ✂️

Comment (maethu) [inline ftw/theming/utils.py]: @jone done.

**Comment by mbaechtold** [comment]
@jone, do you have an idea what is causing the `ReadConflictError` in `test_cachekey_refreshes_when_navroot_changes`?

**Comment by maethu** [comment]
@jone This is now ready!

**Pull request merged**